src/SqlStreamStore.TestUtils/Postgres/PostgresContainer.cs

```diff
@@ -58,7 +54,7 @@ await Policy
             Username = "postgres",
             Host = "127.0.0.1",
             Pooling = true,
-            MaxPoolSize = 50
+            MaxPoolSize = 500
```

Comment:
@ArneSchoonvliet Shouldn't we keep a limit of at least 100?
Since it now has the default settings of pgsql?